src/main/java/org/terasology/module/inventory/ui/ContainerScreen.java

```diff
@@ -29,15 +34,29 @@ public EntityRef get() {
         });
         inventory.setCellOffset(10);
 
+        UILabel containerTitle = find("containerTitle", UILabel.class);
         containerInventory = find("container", InventoryGrid.class);
+
+        EntityRef characterEntity = localPlayer.getCharacterEntity();
+        CharacterComponent characterComponent = characterEntity.getComponent(CharacterComponent.class);
```

Comment:
Moving this out of the `get` might be fine, but we should be aware of the difference in semantics here. 

I'm not 100% sure when the binding is evaluated, but I think this happens more often than the container screen is initialized. 
I don't expect the `characterEntity` to change during the life-time of this container screen, but I'm wondering whether we should fetch the `characterComponent` each time rather than just once in the beginning.